docs/source/tutorials/fine_tune_llama_7b.mdx

```diff
@@ -0,0 +1,356 @@
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Fine-tune and Test Llama 2 7B on AWS Trainium 
+
+This tutorial will teach how to fine-tune open LLMs like [Llama 2](https://huggingface.co/meta-llama/Llama-2-7b-hf) on AWS Trainium. In our example, we are going to leverage Hugging Face https://huggingface.co/docs/optimum-neuron/index, [Transformers](https://huggingface.co/docs/transformers/index) and https://huggingface.co/docs/datasets/index.
+
+You will learn how to:
+
+1. [Setup AWS environment](#1-setup-aws-environment)
+2. [Load and process the dataset](#2-load-and-prepare-the-dataset)
+3. [Fine-tune Llama on AWS Trainium using the `NeuronTrainer`](#3-fine-tune-llama-on-aws-trainium-using-the-neurontrainer)
+4. [Evalaute and test fine-tuned Llama model](#4-evalaute-and-test-fine-tuned-llama-model)
+
+## Quick intro: AWS Trainium
+
+[AWS Trainium (Trn1)](https://aws.amazon.com/de/ec2/instance-types/trn1/) is a purpose-built EC2 for deep learning (DL) training workloads. Trainium is the successor of [AWS Inferentia](https://aws.amazon.com/ec2/instance-types/inf1/?nc1=h_ls) focused on high-performance training workloads. Trainium has been optimized for training natural language processing, computer vision, and recommender models used. The accelerator supports a wide range of data types, including FP32, TF32, BF16, FP16, UINT8, and configurable FP8.
+
+The biggest Trainium instance, the `trn1.32xlarge` comes with over 500GB of memory, making it easy to fine-tune ~10B parameter models on a single instance. Below you will find an overview of the available instance types. More details [here](https://aws.amazon.com/de/ec2/instance-types/trn1/#Product_details):
+
+| instance size                 | accelerators | accelerator memory | vCPU | CPU Memory | price per hour |
+| ----------------------------- | ------------ | ------------------ | ---- | ---------- | -------------- |
+| trn1.2xlarge                  | 1            | 32                 | 8    | 32         | \$1.34         |
+| trn1.32xlarge                 | 16           | 512                | 128  | 512        | \$21.50        |
+| trn1n.32xlarge (2x bandwidth) | 16           | 512                | 128  | 512        | \$24.78        |
+
+_Note: This tutorial was created on a trn1.32xlarge AWS EC2 Instance._
+
+## 1. Setup AWS environment
+
+In this example, we will use the `trn1.32xlarge` instance on AWS with 16 Accelerator, including 32 Neuron Cores and the [Hugging Face Neuron Deep Learning AMI](https://aws.amazon.com/marketplace/pp/prodview-gr3e6yiscria2). The Hugging Face AMI comes with all important libraries, like Transformers, Datasets, Optimum and Neuron packages pre-installed this makes it super easy to get started, since there is no need for environment management.
```

Comment:
```suggestion
In this example, we will use the `trn1.32xlarge` instance on AWS with 16 Accelerator, including 32 Neuron Cores and the [Hugging Face Neuron Deep Learning AMI](https://aws.amazon.com/marketplace/pp/prodview-gr3e6yiscria2). The Hugging Face AMI comes with all important libraries, like Transformers, Datasets, Optimum and Neuron packages pre-installed. This makes it super easy to get started, since there is no need for environment management.
```